frontend/common/stores/project-store.js

```diff
@@ -161,6 +161,21 @@ const store = Object.assign({}, BaseStore, {
       return Promise.resolve(store.getEnvironmentIdFromKey(apiKey))
     })
   },
+  getMaxFeaturesAllowed: () => {
+    return store.model && store.model.max_features_allowed
```

Comment:
Can these be null? If so, what happens?

Reply:
This should not be `null` since, by default, when creating a project a max is created for these fields:
```
max_segments_allowed
max_features_allowed
max_segment_overrides_allowed
```
Also, they should not be nullable as reflected in the OpenAPI doc.

![image](https://github.com/Flagsmith/flagsmith/assets/41410593/13a60166-5890-4a46-8b69-f492387a3b68)